### ISSUE TITLE
Marker comes after measurement if they are the same length

### ISSUE THREAD

**Issue by grahamrow** [BBN-Q/QGL#195]
```python
q1.measure_chan.pulse_params['length'] = 1000e-9
q1.measure_chan.trig_chan.pulse_params['length'] = 100e-9
plot_pulse_files(RabiAmp(cl["q1"], np.linspace(-1, 1, 11)), time=True)
```
![screen shot 2019-02-27 at 5 13 03 pm](https://user-images.githubusercontent.com/1561741/53526783-1822c680-3ab3-11e9-9803-354061a04e98.png)
```python
q1.measure_chan.pulse_params['length'] = 100e-9
```
Here we see the issue: instead of being simultaneous, the marker is AFTER the measure pulse. A whole bunch of the QGL test waveforms fall into this scenario. Could be a plotting issue, need to investigate further.
![screen shot 2019-02-27 at 5 13 10 pm](https://user-images.githubusercontent.com/1561741/53526788-1a852080-3ab3-11e9-8598-d76535ce6804.png)
```python
q1.measure_chan.trig_chan.pulse_params['length'] = 150e-9
```
![screen shot 2019-02-27 at 5 13 15 pm](https://user-images.githubusercontent.com/1561741/53526800-2375f200-3ab3-11e9-8ee5-7fb7c1aacb69.png)

**Comment by dieris** [comment]
I think it's real, but I also thought I had fixed it here: https://github.com/BBN-Q/QGL/pull/189

**Comment by grahamrow** [comment]
Indeed. Cherry-picked fix from #189. Closing.
